docs/data-sources/1000Genomes.mdx

```diff
@@ -0,0 +1,137 @@
+```
+
+The ancestral allele, if it exists, is the first value in the pipe separated AA fields (the Indel specific REF, ALT, IndelType fields are ignored).
+
+We parse the VCF file and extract the following fields from INFO:
+* AA
+* AC
+* AN
+* EAS_AN
+* AMR_AN
+* AFR_AN
+* EUR_AN
+* SAS_AN
+* EAS_AC
+* AMR_AC
+* AFR_AC
+* EUR_AC
+* SAS_AC
+
+
+#### Conflict Resolution
+We have observed conflicting allele frequency information in the source. Take the following example:
+```scss
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
+1 20505705 rs35377696 C CTCTG,CTG,CTGTG 100 PASS AC=46,1513,152;AF=0.0091853,0.302117,0.0303514;
+1 20505705 rs35377696 C CTG 100 PASS AC=4;AF=0.000798722;
+```
+That is, the variant 1-20505705-C-CTG has conflicting entries. To get an idea of how frequently we observe this, here is a table summarizing ChrX and all chromosomes. Note that almost all such entries are found in ChrX.
+
+|Chromosome | \# of alleles    | \# of conflicting alleles | percentage|
+|:---------:|:-----------------|:--------------:|:-----------------|
+| chrX      |       834800     |       2733     |           0.33%  |
+| Total     |       21413098   |       2743     |           0.013% |
+
+**Currently**, we removed the allele frequency of the conflicting allele (i.e., insertion TG in the example) but keep allele frequencies of all other alleles in the VCF line.
+
+**Potential Alternate Solutions**
+* Remove all alleles that are contained in the vcf lines which have conflicting allele. (Recommended by 1000 genome group Holly Zheng-Bradley, 7/29/2015)
+* Recalculate the allele frequency for the conflicting allele.
+* Pick the allele frequency that has the highest data support.
+
+## Download URL
+
+[GRCh37](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/)
+[GRCh38](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000_genomes_project/release/20190312_biallelic_SNV_and_INDEL/)
+
+## JSON Output
+
+<JSONSNV />
+
+## Structural Variants
+### VCF File Parsing
+
+1000 Genomes contains 5 types of structural variants:
+* CNV
+* DEL
+* DUP
+* INS
+* INV
+
+Since data of 1000 genomes is provided in VCF format, we assume that the coordinates follow the vcf format, i.e., there is a padding base for symbolic alleles. So all the interval can be interpreted as [BEGIN+1, END].
+Similarly, for all other variant types except insertion, END is far larger than BEGIN. The distribution of BEGIN and END for insertions is summarized below.
+
+**Insertion issues**
+* END = BEGIN for 6/165
+* END = BEGIN+2 for 93/165
+* END = BEGIN+3 for 11/165
+* END = BEGIN+4 for 11/165
+* END – BEGIN range from 5 to 1156 for others.
+
+#### Example
```

Comment:
Generally I like to start with the example first before talking about how to parse it. Also Parsing and Example are generally at the same level.

Reply:
I removed the Example tag. I started with the vcf line and described what is don.